docs/api_wrapper.md

```diff
@@ -0,0 +1,48 @@
+# APIWrapper support
+
+Create an API session that facilitates usage of the other APIs. 
+
+The api_wrapper class is a wrapper for all of the supported API methods in the Python Client API. This makes it easier to use the supported API methods without having to manually pass around a ServerContext object.
+
+### Using the APIWrapper class
+
+The APIWrapper class is imported from api_wrapper.py:
+
+```python
+from labkey.api_wrapper import APIWrapper
+```
+
+It includes the following arguments:
+
+**Domain**
+- This is the base URL for a LabKey Server instance.
+- Example: 'www.labkey.org'
+
+**Container_path**
+- This is the path to the targeted project, folder, or subfolder in a LabKey Server instance. It is similar to a file path. Only absolute paths can be inputted.
+- Example: 'Project/Folder/Subfolder'
+
+**context_path** 
+- The default value is None. Depending on how the LabKey Server instance is implemented, it may be necessary to include a value for the context_path argument. If your LabKey Server instance has text after the base URL, that is the context path. 
+- Example: If your home project has a URL such as https://labkey.org/contextpath/home/project-begin.view, then the context path is 'contextpath'.
+
+**use_ssl**
+- The default value is True. This should be set to True if your server is configured to use SSL. If you are not sure if your server uses SSL, refer to any URL for accessing your server. Servers using SSL will have a URL that begins with https instead of http. LabKey Sample Manager-only clients must have this argument set to True.
+
+**verify_ssl**
+- The default value is True. This argument is only relevant for scripts that use webdav-related features in the LabKey Python API. It forces your script to check if the targeted server is configured to use SSL or not.
```

Comment:
```suggestion
- The default value is True. This argument toggles whether or not the SSL certificate is validated when attempting to connect to a server. This flag is useful when you are connecting to a development server with a self-signed SSL certificate, which would otherwise cause a failure. You should never disable this flag if you are connecting to a production server with a proper SSL certificate.
```